Makefile.js

```diff
@@ -705,9 +718,20 @@ target.gensite = function(prereleaseVersion) {
                 } else {
                     title = "Documentation";
                 }
-                text = `---\ntitle: ${title}\nlayout: doc\n---\n<!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->\n\n${text}`;
+                editLink = `https://github.com/eslint/eslint/edit/master/${filePath}`;
             }
 
+            text = [
+                "---",
+                `title: ${title}`,
+                "layout: doc",
+                editLink ? `edit_link: ${editLink}` : "",
```

Comment:
Nitpick: Will `editLink` ever be falsy here? It seems like it gets assigned on every code path.

Reply:
You’re right, removed the conditional.